mappings/1.8/net/minecraft/entity/living/mob/hostile/ZombiePigmanEntity.mapping

```diff
@@ -3,5 +3,6 @@ CLASS net/minecraft/C_45ultkugn net/minecraft/entity/living/mob/hostile/ZombiePi
 	FIELD f_39sxgdfww angerValue I
 	FIELD f_50ogqmmus angerSoundDelay I
 	FIELD f_78jklzenu ATTACKING_SPEED_BOOST_ID Ljava/util/UUID;
+	FIELD f_95iazvwia attackerUUID Ljava/util/UUID;
```

Comment:
As per conventions elsewhere, `uuid` should be considered a single word, and should be capitalized as `Uuid`.